portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,36 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1 id ="title">Santiago Minga's Portfolio</h1>
+      <div id="gallery">
+        <img id="me" src="images/thisisme.png" />
+      </div>
+      <p class= "text1">Hello everyone! You've seen my name by now, but allow me to introduce myself.
+        I'm a 19 years old Computer Science student down at Tecnologico de Monterrey, 
+        city where I currently live in, even though I'm originally from Ecuador. I love to draw, specially
+        with ink, pencils and pens, mostly doing black and white pieces, let me show you an example:  </p>
+        <div id="gallery">
+        <img class = "img" src="images/art1.png" /> 
+        <img class = "img" src="images/art2.png" /> 
+        <img class = "img" src="images/art3.png" /> 
+        </div>
+      <p class="text1"> Anyways, heres <a href="https://github.com/SantiM1">my GitHub</a>. 
+        Right now there's not much, but I'm hoping it'll grow exponentially soon, it's quite a shame my most 
+        impressive project(in my opinion) is currently private due to a privacy agreements but let me show you some the things 
+        I'm allowed to, like a small demo of the working product which was a heatmap implemented in Unity.</p>
+        <div id="gallery">
+          <img id="gif" src="images/project.gif" />
+        </div>
+      <p class ="text1">Most recently I've been working on an app mockup for an online library concept, however it is in Spanish.
+         You can interact with the demo by clicking on it how you would with a normal phone and pressing R to restart it. </p>
+      <div id="gallery">
+        <iframe style="border: 1px solid rgba(0, 0, 0, 0.1);" width="300" height="500" 
+        src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Fproto%2FOMYCe2rY5U9dfFR27V6wVp%2FTutecaDemo3%3Fpage-id%3D0%253A1%26node-id%3D2%253A2%26viewport%3D14%252C-227%252C0.43%26scaling%3Dscale-down%26starting-point-node-id%3D7%253A50" 
+        allowfullscreen></iframe>
+      </div>
+      <p class="text1">Click <a href="/hello">here</a> to see content created by server code.</p>
+      <p class="text1">Click here to get a random fact about me:</p>
+      <button onclick="addRandomFact()">:D</button>
       <div id="greeting-container"></div>
```

Comment:
Since you are changing "greeting" to "fact", you should probably update the id here.

Reply:
Oh, true. Thank you, I'll be on the lookout for stuff like that in the future.